frontend/src/modules/addonpromotion/components/AddonPromotion.js

```diff
@@ -68,6 +68,7 @@ export class AddonPromotionBase extends React.Component<InternalProps, State> {
         // Add-On installed
         if (
             this.state.installed ||
+            // @ts-expect-error
```

Comment:
I haven't figured out how to amend the global type in TypeScript, let alone doing so in a way that wouldn't surprise flow.